pkg/api/dashboard.go

```diff
@@ -154,6 +174,14 @@ func (hs *HTTPServer) GetDashboard(c *models.ReqContext) response.Response {
 			return response.Error(500, "Error while loading library panels", err)
 		}
 	}
+	var trimedJson simplejson.Json
+	if trimDefaults {
```

Comment:
I would rather add the feature toggle check here in the API layer since then it would not even call the `LoadSchemaService` if disabled and completely skip the whole thing in case there are unexpected bugs :smile:

Reply:
i was thinking about it, but ngalert is doing more in the service. :) will fix it.